### ISSUE TITLE
Wrong limit calculation

### ISSUE THREAD

**Issue by rtoy** [rtoy/maxima#3029]
Imported from SourceForge on 2024-07-07 09:33:57
Created by **[kcrisman](https://sourceforge.net/u/kcrisman/)** on 2018-08-15 13:00:59
Original: https://sourceforge.net/p/maxima/bugs/3459

---

```
Maxima 5.41.0 http://maxima.sourceforge.net
using Lisp ECL 16.1.2
Distributed under the GNU Public License. See the file COPYING.
Dedicated to the memory of William Schelter.
The function bug_report() provides bug reporting information.
(%i1) limit(x / (x+2^x+cos(x)),x,-inf)       ;
(%o1)                                 und
```
Sympy says it is equal to 1, and a plot suggests this is correct.

Upstream:  https://trac.sagemath.org/ticket/26060

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-07 09:33:59
Created by **[robert_dodier](https://sourceforge.net/u/robert_dodier/)** on 2018-08-18 19:07:23
Original: https://sourceforge.net/p/maxima/bugs/3459/#8663

---

- **labels**:  --> limit

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-07 09:34:02
Created by **[dgildea](https://sourceforge.net/u/dgildea/)** on 2018-08-30 21:37:31
Original: https://sourceforge.net/p/maxima/bugs/3459/#100f

---

fixed by [0c5c0187338060761746fbd0821ce5f28b3245cd]
